### PR TITLE
Add links to open issue and create pull request in the whitepaper

### PR DESCRIPTION
**Pull request by wjsvec**
#305

### CODE REVIEW

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Comment by wjsvec** [comment]
Hey @taylor, could you please take a moment to review my GitHub PR? Thanks!🥺

**Review by lixuna — APPROVED** [review]
lgtm

**Review by jeffsaelens — APPROVED** [review]
lgtm

File: doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ v1.0 - December 4, 2023
...
+This document is a product of the initial joint work of several Communication Service Providers (CSPs) who are active in the Cloud Native Computing Foundation (CNCF)’s Cloud Native Network Function Working Group (CNF WG), NGMN Alliance, and projects like Linux Foundation (LF) Europe’s Sylva and Linux Foundation Networking (LFN) Anuket project. It is a draft that has been published to invite feedback from other CSPs and motivate discussion and improvements from the broader telecommunication industry. We hope that through public discourse we can make the document more complete, relevant, and ready for final release. If you would like to contribute to the discussion and document, please feel free to [open an issue](https://github.com/cncf/cnf-wg/issues/new/choose) or [create a pull request](https://github.com/cncf/cnf-wg/compare).
```
Comment (lixuna) [inline doc/whitepaper/Accelerating_Cloud_Native_in_Telco.md]:
```suggestion
This document is a product of the initial joint work of several Communication Service Providers (CSPs) who are active in the Cloud Native Computing Foundation (CNCF)’s Cloud Native Network Function Working Group (CNF WG), NGMN Alliance, and projects like Linux Foundation (LF) Europe’s Sylva and Linux Foundation Networking (LFN) Anuket project. It is a draft that has been published to invite feedback from other CSPs and motivate discussion and improvements from the broader telecommunication industry. We hope that through public discourse we can make the document more complete, relevant, and ready for final release. If you would like to contribute to the discussion and document, please feel free to [open an issue](https://github.com/lfn-cnti/bestpractices/issues/new) or [create a pull request](https://github.com/lfn-cnti/bestpractices/compare).
```

**Review by lixuna — COMMENTED** [review]
Github repos have been merged to new org. URLs updated to reflect new location > https://github.com/lfn-cnti/bestpractices/

**Review by taylor — COMMENTED** [review]
lgtm

**Pull request merged**